src/Parser/AfflictionWarlock/Modules/Features/CorruptionUptime.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+
+import Module from 'Parser/Core/Module';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS';
+import SpellIcon from 'common/SpellIcon';
+import { formatPercentage } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class CorruptionUptime extends Module {
+  static dependencies = {
+    enemies: Enemies,
+  };
+
+  suggestions(when){
+    const corruptionUptime = this.enemies.getBuffUptime(SPELLS.CORRUPTION_DEBUFF.id) / this.owner.fightDuration;
+    if(this.owner.selectedCombatant.hasBuff(SPELLS.WARLOCK_AFFLI_T20_2P_BONUS.id)) {
+      when(corruptionUptime).isLessThan(0.95)
+        .addSuggestion((suggest, actual, recommended) => {
+          return suggest('Your Corruption uptime can be improved. Try to pay more attention to your Corruption on the boss, especially since you have the T20 2-piece set bonus.')
```

Comment:
My English is sometimes not top-notch so you are correct, that sounds a bit more human. Adding a link to the buff shouldn't be a problem, I'll have a look into that.